collect_app/src/androidTest/java/org/odk/collect/android/support/pages/OpenSourceLicensesPage.java

```diff
@@ -0,0 +1,17 @@
+package org.odk.collect.android.support.pages;
+
+import androidx.test.rule.ActivityTestRule;
+
+class OpenSourceLicensesPage extends Page<OpenSourceLicensesPage> {
+
+    OpenSourceLicensesPage(ActivityTestRule rule) {
+        super(rule);
+    }
+
+    @Override
+    public OpenSourceLicensesPage assertOnPage() {
+        waitForText("Open Source Licenses");
```

Comment:
This was failing for me locally. Looks like the page takes a second or two to load but that the standard Espresso wait syncs (on background threads etc) don't work. Probably because of the `WebView` involved I'd imagine.